R/calc_aggregate_counts.R

```diff
@@ -21,25 +31,43 @@
 #' @export
 
 calc_aggregate_counts <- function(
-    window = 31, ucla_only = FALSE, state = FALSE, collapse_vaccine = TRUE){
+    window = 31, ucla_only = FALSE, state = FALSE, collapse_vaccine = TRUE,
+    all_dates = FALSE, week_grouping = TRUE){
+
+    round_ <- ifelse(week_grouping, "week", "month")
 
     to_report <- c(datasets::state.name, "Federal", "ICE")
```

Comment:
This means we lose DC, right?

Reply:
yes DC will not be incorporated into this need to update the docs